### PR TITLE
feat(playground): auto-inject ion-content and ion-app

### PR DESCRIPTION
**Pull request by sean-perkins**
Introduces `includeIonContent` property to the `Playground` component. This option will automatically inject an `ion-content` container into the examples, so that they do no need to be manually specified. Implementations can opt-out of this behavior if they need to control their own container (modal does this).

Migrated existing playground examples to remove manually specifying the container.

Angular examples now include an `example.component.ts` and `example.component.html` for the custom snippet content.

Components modified:

- [x] `ion-accordion`
- [x] `ion-alert`
- [x] `ion-breadcrumbs`
- [x] `ion-datetime`
- [x] `ion-loading`
- [x] `ion-modal`
- [x] `ion-picker`
- [x] `ion-popover`
- [x] `ion-range`
- [x] `ion-select`

I went through each example and opened it in Stackblitz. While doing so, I found minor issues with examples pre-Typescript and a few inconsistencies across examples. I fixed those issues while reviewing.

### CODE REVIEW

**Review by amandaejohnston — APPROVED** [review]

File: static/usage/accordion/customization/advanced-expansion-styles/react/react-ts.md [hunk 1]
```diff
@@ -1,17 +1,12 @@
...
-  IonItem, 
-  IonLabel
-} from '@ionic/react';
+import { IonAccordion, IonAccordionGroup, IonItem, IonLabel } from '@ionic/react';
 
 import './main.css';
 
 function Example() {
   return (
-    <IonAccordionGroup expand="inset">
```
[the remark this answers is not in the document]
Comment (sean-perkins) [reply static/usage/accordion/customization/advanced-expansion-styles/react/react-ts.md:14]: Yes, the other examples (JS, Angular & Vue) do not have this option specified. It changes the visual output of the Stackblitz example. I removed to be consistent with the demo & examples. 

Likely was just the result of copy & paste originally.

File: static/usage/datetime/theming/index.md [hunk 2]
```diff
@@ -7,23 +7,24 @@ import vue from './vue.md';
...
-      }
-    }
-  }} 
+        'src/app/example.component.html': angularHTML,
+        'src/styles.css': angularCSS,
+      },
+    },
+  }}
   src="usage/datetime/theming/demo.html"
+  includeIonContent={false}
```
Comment (liamdebeasi) [inline static/usage/datetime/theming/index.md:29]: Why do we not include ion-content here? Also I noticed that ion-app does not get injected in either Angular or JS examples.

File: static/usage/modal/controller/index.md [hunk 3]
```diff
@@ -34,4 +34,5 @@ import angular_modal_example_component_html from './angular/modal-example_compon
   }}
   src="usage/modal/controller/demo.html"
   devicePreview
+  includeIonContent={false}
```
Comment (liamdebeasi) [inline static/usage/modal/controller/index.md:37]: ion-app is not injected in the Angular and JS examples

File: static/usage/modal/inline/is-open/index.md [hunk 4]
```diff
@@ -14,11 +14,12 @@ import angular_app_component_ts from './angular/app_component_ts.md';
...
-        'src/app/app.component.html': angular_app_component_html,
-        'src/app/app.component.ts': angular_app_component_ts,
+        'src/app/example.component.html': angular_example_component_html,
+        'src/app/example.component.ts': angular_example_component_ts,
       },
     },
   }}
   src="usage/modal/inline/is-open/demo.html"
   devicePreview
+  includeIonContent={false}
```
Comment (liamdebeasi) [inline static/usage/modal/inline/is-open/index.md:24]: ion-app is not injected in the Angular and JS examples

File: static/usage/modal/sheet/background-content/index.md [hunk 5]
```diff
@@ -15,12 +15,13 @@ import angular_app_component_css from './angular/app_component_css.md';
...
-        'src/app/app.component.css': angular_app_component_css,
+        'src/app/example.component.html': angular_example_component_html,
+        'src/app/example.component.ts': angular_example_component_ts,
+        'src/app/example.component.css': angular_example_component_css,
       },
     },
   }}
   src="usage/modal/sheet/background-content/demo.html"
   devicePreview
+  includeIonContent={false}
```
Comment (liamdebeasi) [inline static/usage/modal/sheet/background-content/index.md:26]: ion-app is not injected in the Angular and JS examples

File: static/usage/modal/sheet/basic/index.md [hunk 6]
```diff
@@ -14,4 +14,5 @@ import angular from './angular.md';
   }}
   src="usage/modal/sheet/basic/demo.html"
   devicePreview
+  includeIonContent={false}
```
Comment (liamdebeasi) [inline static/usage/modal/sheet/basic/index.md:17]: ion-app is not injected in the Angular and JS examples

File: src/components/global/Playground/README.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+    vue,
+  }}
+/>;
+```
+
+The file location is respective to the Stackblitz demo project. If you are unsure of the exact path, refer to the `stackblitz.utils.ts` method for each respective target. The path specified in `sdk.openProject` will coincide with the value used in the `files` configuration.
+
+## Ejecting from IonApp/IonContent Boilerplate
+
+By default, Playground examples will automatically include the following template wrapper each component example when opened in Stackblitz.
```
Comment (liamdebeasi) [inline src/components/global/Playground/README.md:53]:
```suggestion
By default, Playground examples will automatically include the following template wrapper for each component example when opened in Stackblitz.
```

**Review by liamdebeasi — APPROVED** [review]

**Pull request merged**